control_panel_api/elasticsearch.py

```diff
@@ -0,0 +1,26 @@
+from elasticsearch import Elasticsearch
+from elasticsearch_dsl import Q, Search
+from elasticsearch_dsl.query import Range
+
+
+class ElasticSearch(object):
```

Comment:
This module now contains `Elasticsearch` and `ElasticSearch` classes. It could potentially result in difficult to catch bugs if someone makes a quite likely typo. Maybe `import Elasticsearch as _Elasticsearch` or import the module and use `self.client = elasticsearch.Elasticsearch()` or change the name of this class.